adcc/backends/psi4.py

```diff
@@ -39,7 +39,19 @@ def __init__(self, wfn):
 
     @cached_property
     def electric_dipole(self):
-        return [-np.asarray(comp) for comp in self.mints.ao_dipole()]
+        return [-1.0 * np.asarray(comp) for comp in self.mints.ao_dipole()]
+
+    @cached_property
+    def magnetic_dipole(self):
+        # TODO: Gauge origin?
+        return [
+            0.5 * np.asarray(comp)
+            for comp in self.mints.ao_angular_momentum()
+        ]
+
+    @cached_property
+    def momentum(self):
```

Comment:
Why -1/2? I only know -i\hbar\nabla (like [here](https://en.wikipedia.org/wiki/Momentum_operator)).
`nabla` would be fine by me, because the only difference is the sign, which is, more or less random depending on the underlying integral library... The most important thing is to get the correct oscillator strengths (independent of the sign), and rotatory strengths.

Reply:
Because I'm stupid and though about kinetic energy. Forget the 1/2. But still momentum implies there would be an `i` ... so my argument still stands. Just go for `nable` :smile:

Reply:
Done.